### PR TITLE
Add german translation of Readme

### PR DESCRIPTION
**Pull request by christian-draeger**

### CODE REVIEW

**Comment by anuraghazra** [comment]
Wow this is super awesome @christian-draeger Thank you very much. :100:

File: readme_de.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+Füge diesen Code in deine Readme-Datei ein und passe die Links an.
+
+Endpunkt: `api/top-langs?username=anuraghazra`
+
+```md
+[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
+```
+
+### Verberge Programmiersprachen unter einem bestimmten Schwellwert
```
Comment (anuraghazra) [inline readme_de.md:154]: btw i really don't know german but is the word "Programmiersprachen" means "Languages" or "Programming languages" ???

If its "Programming languages" then we need to change to "Languages"

Comment (saitho) [reply readme_de.md:154]: Yes, it is "Programming languages"

Comment (anuraghazra) [reply readme_de.md:154]: It should be just "Top Languages" @christian-draeger

File: readme_de.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+</a>
+<a href="https://github.com/anuraghazra/convoychat">
+  <img align="left" src="https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=convoychat" />
+</a>
+```
+
+## Betreibe es auf deiner eigenen Vercel-Instanz
+
+Da die GitHub API nur 5tsd Aufrufe pro Stunde zulässt, kann es passieren, dass meine `https://github-readme-stats.vercel.app/api` dieses Limit erreicht.
+If you host it on your own Vercel server, then you don't have to worry about anything. Click on the deploy button to get started!
```
Comment (saitho) [inline readme_de.md:218]: Missing translation here.

```suggestion
Wenn du es auf deinem eigenen Vercel-Server hostest, brauchst du dich darum nicht zu kümmern. Klicke auf den Deploy-Button um loszulegen!
```

File: readme_de.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+Hinweis: Seit [#58](https://github.com/anuraghazra/github-readme-stats/pull/58) sollte es möglich sein mehr als 5tsd Aufrufe pro Stunde ohne Downtimes zu verkraften :D
+
+[![Deploy to Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/anuraghazra/github-readme-stats)
+
+<details>
+ <summary>Guide on setting up Vercel</summary>
```
Comment (saitho) [inline readme_de.md:225]: Missing translation here.

```suggestion
 <summary>Anleitung zum Einrichten von Vercel</summary>
```

File: readme_de.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+1. Klick `Import Project`  
+   ![](https://files.catbox.moe/qckos0.png)
+1. Klick `Import Git Repository`  
+   ![](https://files.catbox.moe/pqub9q.png)
+1. Wähle root und füge eine Umgebungsvariable namens PAT_1 (siehe Abbildung) die als Wert deinen persönlichen Access Token (PAT) hat hinzu, den du einfach [hier](https://github.com/settings/tokens/new) erzeugen kannst (lasse alles wie es ist, vergebe einen beliebigen Namen)
+   ![](https://files.catbox.moe/0ez4g7.png)
+1. Klicke deploy, und das wars. Besuche deine domains um die API zu benutzen!
+</details>
+
+## :sparkling_heart: Support the project
```
Comment (saitho) [inline readme_de.md:244]:
```suggestion
## :sparkling_heart: Unterstütze das Projekt
```

**Comment by christian-draeger** [comment]
@saitho thx for proof reading it and the additions. Was reading it 3 times and still missed some parts 😅

**Comment by anuraghazra** [comment]
Hi @christian-draeger thanks for the contribution :)

**Pull request merged**